lightfm/_lightfm_fast.pyx.template

```diff
@@ -800,14 +800,15 @@ def fit_warp(CSRMatrix item_features,
     """
 
     cdef int i, no_examples, user_id, positive_item_id, gamma
-    cdef int negative_item_id, sampled, row
+    cdef int negative_item_ix, negative_item_id, sampled, row
     cdef double positive_prediction, negative_prediction
     cdef double loss, MAX_LOSS
     cdef flt weight
     cdef flt *user_repr
     cdef flt *pos_it_repr
     cdef flt *neg_it_repr
     cdef unsigned int[::1] random_states
+    cdef unsigned int[::1] menu_recipe_ids
```

Comment:
Instead of typing the variable here, for the next ticket I move this into the method arguments where it is passed down from within the _run_epoch method in lightfm.py